web/app/components/header/nav.hbs

```diff
@@ -43,6 +43,14 @@
         @icon="file-plus"
         class="create-draft-button"
       />
```

Comment:
why two same buttons ? what is the difference between them

Reply:
Yes they are duplicate thanks for noticing I will change it!

Reply:
Yes they are duplicate thanks for noticing I will change it!